django/conf/locale/cs/formats.py

```diff
@@ -14,8 +14,10 @@
 # The *_INPUT_FORMATS strings use the Python strftime format syntax,
 # see https://docs.python.org/library/datetime.html#strftime-strptime-behavior
 DATE_INPUT_FORMATS = [
-    '%d.%m.%Y', '%d.%m.%y',     # '05.01.2006', '05.01.06'
-    '%d. %m. %Y', '%d. %m. %y',  # '5. 1. 2006', '5. 1. 06'
+    '%d.%m.%Y',  # '05.01.2006'
+    '%d.%m.%y',  # '05.01.06'
+    '%d. %m. %Y',  # '5. 1. 2006'
+    '%d. %m. %y',  # '5. 1. 06'
     # '%d. %B %Y', '%d. %b. %Y',  # '25. October 2006', '25. Oct. 2006'
```

Comment:
Can we remove or reformat this comments here and in other languages ?

Reply:
These comment are not affected by `black`. I don't see much value in changing them, see https://github.com/django/django/pull/15379#discussion_r796034343.

Reply:
I understand that it is an extra job, but I just thought that you could take advantage of this PR to clean up even those lines commented on for some time, which someone could uncomment in the future without breaking them correctly.

Reply:
Agreed that it would be clearer to rearrange the comment. Also black won't fix the quotes in commented code.

```suggestion
    # "%d. %B %Y",  # '25. October 2006'
    # "%d. %b. %Y",  # '25. Oct. 2006'
```

I know we normally avoid changing unrelated code, but I would consider this part of changing the hanging indentation.

Reply:
OK, I reformatted comments.